.travis.yml

```diff
@@ -17,19 +14,50 @@ env:
     - GDALBUILD=$HOME/gdalbuild
     - PROJBUILD=$HOME/projbuild
     - PROJVERSION="6.0.0"
-    
-  matrix:
-
-    - GDALVERSION="2.2.4"
-    - GDALVERSION="2.3.3"
-    - GDALVERSION="2.4.1"
-    - GDALVERSION="3.0.0"
-    - GDALVERSION="trunk"
 
 matrix:
+  include:
+    - python: "2.7"
+      env:
+      - GDALVERSION="2.2.4"
+
+    - python: "3.5"
+      env:
+      - GDALVERSION="2.2.4"
+
+    - python: "3.6"
+      env:
+      - GDALVERSION="2.2.4"
+
+    - python: "3.7"
+      env:
+      - GDALVERSION="2.2.4"
+
+    - python: "3.7"
+      env:
+      - GDALVERSION="2.3.3"
```

Comment:
@mwtoews I like this approach of making the matrix more sparse. I think we can take it a little further. Python 3.7 hasn't made it out into distros very far yet, and so I think that any users are likely to be using nearly the cutting edge of GDAL. What would you think about dropping GDAL versions < 2.4 for Python 3.7?